dll/win32/msvidc32/msvideo1.c

```diff
@@ -538,7 +538,11 @@ static LRESULT CRAM_GetInfo( const Msvideo1Context *info, ICINFO *icinfo, DWORD
 
     icinfo->dwSize = sizeof(ICINFO);
     icinfo->fccType = ICTYPE_VIDEO;
+#ifdef __REACTOS__
+    icinfo->fccHandler = info ? info->dwMagic : MSVC_MAGIC;
+#else
     icinfo->fccHandler = info ? info->dwMagic : CRAM_MAGIC;
+#endif
```

Comment:
Please double-check whether this is correct change. A VfW codec could support several FOURCC's, but usually exposes just one for compression. (I'm pretty sure msvidc32 should expose `CRAM`)

Reply:
Reverted.

Reply:
Partially fixed.